crud/pom.xml

```diff
@@ -73,32 +82,52 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.apache.tomcat</groupId>
-            <artifactId>tomcat-catalina</artifactId>
-            <version>8.0.9</version>
+            <groupId>org.jboss.arquillian.container</groupId>
+            <artifactId>arquillian-jetty-embedded-7</artifactId>
+            <version>1.0.0.CR3</version>
```

Comment:
Using Jetty with Arquillian for two main reasons:

1. It's a real servlet container, so we can test the production web.xml, making sure our endpoints (like the healthcheck) are wired up correctly.
2. It's easily embeddable and quick to start. Embedding JBoss can't really be done, a distribution has to be downloaded, which doesn't integrate well with IDEs... it's a mess.

I tried Tomcat, too, but ran into some classpath issues. Turns out it was unrelated to Tomcat specifically, so we could try Tomcat again, but Jetty serves the same purpose.